spec/features/create_scan_spec.rb

```diff
@@ -55,7 +58,11 @@
 
       click_button 'Send request'
 
-      expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
+      # expect(page).to have_css('h1#dialogTitle', text: 'Request complete')
```

Comment:
I meant the commented `expect` in your test that matches the one above it character for character.  Do we need that commented out `expect`?  I don't quite understand what the H1 has to do w/ the Illiad callback.